README.md

```diff
@@ -226,6 +226,31 @@ Output:
 - Run tests using `bundle exec rake test`. All code should be tested.
 - No code, outside of development and tests needs, should use dependencies. This is a self contained library
 
+### Building
+
+```shell
+gem build cli-ui.gemspec
+```
+
+### Testing Manually
+```shell
+gem install --local cli-ui-*.gem
+irb
+```
```

Comment:
you should be able to do something like `bundle exec ruby test.rb`, no need for building / installing the gem.